collections/open-source-organizations/index.md

```diff
@@ -14,6 +14,7 @@ items:
  - IBM/ibm.github.io
  - Microsoft/microsoft.github.io
  - artsy/artsy.github.io
+ - OSGeo/github.com/OSGeo
```

Comment:
Can this be updated to OSGeo/OSGeo? Thanks!

Reply:
Updated :)